src/go.mod

```diff
@@ -35,12 +35,7 @@ require (
 	github.com/garyburd/redigo v1.6.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-openapi/errors v0.19.2
-	github.com/go-openapi/loads v0.19.3
 	github.com/go-openapi/runtime v0.19.5
-	github.com/go-openapi/spec v0.19.3
-	github.com/go-openapi/strfmt v0.19.3
-	github.com/go-openapi/swag v0.19.5
-	github.com/go-openapi/validate v0.19.3
```

Comment:
These pkgs used by swagger generated APIs and we should not delete them from go.mod.